src/box/lua/upgrade.lua

```diff
@@ -1454,30 +1454,31 @@ end
 --------------------------------------------------------------------------------
 
 local handlers = {
-    {version = mkversion(1, 7, 5), func = upgrade_to_1_7_5},
-    {version = mkversion(1, 7, 6), func = upgrade_to_1_7_6},
-    {version = mkversion(1, 7, 7), func = upgrade_to_1_7_7},
-    {version = mkversion(1, 10, 0), func = upgrade_to_1_10_0},
-    {version = mkversion(1, 10, 2), func = upgrade_to_1_10_2},
-    {version = mkversion(2, 1, 0), func = upgrade_to_2_1_0},
-    {version = mkversion(2, 1, 1), func = upgrade_to_2_1_1},
-    {version = mkversion(2, 1, 2), func = upgrade_to_2_1_2},
-    {version = mkversion(2, 1, 3), func = upgrade_to_2_1_3},
-    {version = mkversion(2, 2, 1), func = upgrade_to_2_2_1},
-    {version = mkversion(2, 3, 0), func = upgrade_to_2_3_0},
-    {version = mkversion(2, 3, 1), func = upgrade_to_2_3_1},
-    {version = mkversion(2, 7, 1), func = upgrade_to_2_7_1},
-    {version = mkversion(2, 9, 1), func = upgrade_to_2_9_1},
-    {version = mkversion(2, 10, 1), func = upgrade_to_2_10_1},
-    {version = mkversion(2, 10, 4), func = upgrade_to_2_10_4},
-    {version = mkversion(2, 10, 5), func = upgrade_to_2_10_5},
-    {version = mkversion(2, 11, 0), func = upgrade_to_2_11_0},
-    {version = mkversion(2, 11, 1), func = upgrade_to_2_11_1},
-    {version = mkversion(3, 0, 0), func = upgrade_to_3_0_0},
-    {version = mkversion(3, 1, 0), func = upgrade_to_3_1_0},
-    {version = mkversion(3, 3, 0), func = upgrade_to_3_3_0},
+    {version = mkversion.new(1, 7, 5), func = upgrade_to_1_7_5},
+    {version = mkversion.new(1, 7, 6), func = upgrade_to_1_7_6},
+    {version = mkversion.new(1, 7, 7), func = upgrade_to_1_7_7},
+    {version = mkversion.new(1, 10, 0), func = upgrade_to_1_10_0},
+    {version = mkversion.new(1, 10, 2), func = upgrade_to_1_10_2},
+    {version = mkversion.new(2, 1, 0), func = upgrade_to_2_1_0},
+    {version = mkversion.new(2, 1, 1), func = upgrade_to_2_1_1},
+    {version = mkversion.new(2, 1, 2), func = upgrade_to_2_1_2},
+    {version = mkversion.new(2, 1, 3), func = upgrade_to_2_1_3},
+    {version = mkversion.new(2, 2, 1), func = upgrade_to_2_2_1},
+    {version = mkversion.new(2, 3, 0), func = upgrade_to_2_3_0},
+    {version = mkversion.new(2, 3, 1), func = upgrade_to_2_3_1},
+    {version = mkversion.new(2, 7, 1), func = upgrade_to_2_7_1},
+    {version = mkversion.new(2, 9, 1), func = upgrade_to_2_9_1},
+    {version = mkversion.new(2, 10, 1), func = upgrade_to_2_10_1},
+    {version = mkversion.new(2, 10, 4), func = upgrade_to_2_10_4},
+    {version = mkversion.new(2, 10, 5), func = upgrade_to_2_10_5},
+    {version = mkversion.new(2, 11, 0), func = upgrade_to_2_11_0},
+    {version = mkversion.new(2, 11, 1), func = upgrade_to_2_11_1},
+    {version = mkversion.new(3, 0, 0), func = upgrade_to_3_0_0},
+    {version = mkversion.new(3, 1, 0), func = upgrade_to_3_1_0},
+    {version = mkversion.new(3, 3, 0), func = upgrade_to_3_3_0},
```

Comment:
These changes look unneeded (and the similar ones below).

Reply:
The default is `version.fromstr` (see metastable od the module), it seems it much more convenient to create objects from strings and not manually with `version.new`

Reply:
OK, got it. Thanks for the clarification!